Doc/c-api/long.rst

```diff
@@ -514,6 +514,17 @@ distinguished from a number.  Use :c:func:`PyErr_Occurred` to disambiguate.
    .. versionadded:: 3.14
 
 
+.. c:function:: PyObject* PyLong_GetInfo(void)
+
+   On success, return a read only :term:`named tuple`, that holds
+   information about Python's internal representation of integers.
+   See :data:`sys.int_info` for description of individual fields.
+
+   On failure, return ``NULL`` with an exception set.
+
+   .. versionadded:: 3.2
```

Comment:
```suggestion
   .. versionadded:: 3.1
```

It was added in commit bd7926478de92a2a0ef4440e1a9ae61b706a80d2.

Reply:
Indeed, I was looking on stable_abi.dat.  Perhaps, this could be corrected too in this pr?

Reply:
Done